publish-docker

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# push the cli4 standalone container image to Dockerhub
+docker push cyberark/conjur-cli4
```

Comment:
Do we need this file still? If so, this needs to be updated. If not, it should be removed.

Reply:
Nope, it's gone.